docs/how-to/rocks/add-internal-user-in-rocks.rst

```diff
@@ -0,0 +1,65 @@
+.. _add_internal_user_in_rocks:
+
+How to add an internal user to a rock
+*************************************
+
+Rockcraft allows you to use :ref:`run-user <rockcraft_yaml_run_user>` to
+specify the user you want to run the rock's service(s) with. Without using that
+field in the ``rockcraft.yaml`` file, the service(s) will be run by the root
+user. However, ``run-user`` only accepts a limited amount of well-known users,
```

Comment:
The supported users are listed in the `run-user` section which was referenced in the first line of this paragraph.

Reply:
Missed that, my bad. Since `_daemon_` is a [Canonical convention](https://discourse.ubuntu.com/t/unifying-user-identity-across-snaps-and-rocks/36469#naming-convention), let's remove "well-known".